cypress/integration/inventory/export.spec.js

```diff
@@ -12,40 +11,60 @@ describe('inventory: exports', () => {
   });
 
   it('C9284 verifies export UUIDs instances', () => {
-    cy.do([
-      InventorySearch.searchByEffectiveLocation(),
-      InventoryActions.open(),
-      InventoryActions.options.saveUUIDs.click()
-    ]);
+    InventorySearch.byEffectiveLocation();
+    InventorySearch.saveUUIDs();
 
     // TODO: think about move it to separate func
-    cy.intercept('/search/instances/ids**').as('getIds');
-    cy.wait('@getIds').then((obj) => {
-      const expectedUUIDs = InventorySearch.getUUIDsFromRequest(obj);
-
-      // Need time for download file TODO: think about how it can be fixed
-      cy.wait(Cypress.env('downloadTimeout'));
-
-      FileManager.findDownloadedFilesByMask('SearchInstanceUUIDs*')
-        .then((downloadedFilenames) => {
-          const lastDownloadedFilename = downloadedFilenames.sort()[downloadedFilenames.length - 1];
-          InventoryActions.verifySaveUUIDsFileName(lastDownloadedFilename);
-
-          FileManager.readFile(lastDownloadedFilename)
-            .then((actualUUIDs) => {
-              InventoryActions.verifySavedUUIDs(actualUUIDs, expectedUUIDs);
-            });
-        });
-    });
+    cy.intercept('/search/instances/ids**')
+      .as('getIds');
+    cy.wait('@getIds')
+      .then((obj) => {
+        const expectedUUIDs = InventorySearch.getUUIDsFromRequest(obj);
+
+        // Need time for download file TODO: think about how it can be fixed
+        cy.wait(Cypress.env('downloadTimeout'));
+
+        FileManager.findDownloadedFilesByMask('SearchInstanceUUIDs*')
+          .then((downloadedFilenames) => {
+            const lastDownloadedFilename = FileManager.getLastDownloadedFileName(downloadedFilenames);
+            InventoryActions.verifySaveUUIDsFileName(lastDownloadedFilename);
+
+            FileManager.readFile(lastDownloadedFilename)
+              .then((actualUUIDs) => {
+                InventoryActions.verifySavedUUIDs(actualUUIDs, expectedUUIDs);
+              });
+          });
+      });
   });
 
   it('C196755 verifies search result counts and selected counts', () => {
```

Comment:
think to keep one test name into testrail and the code as additional option to match(approximately) these entities by people